src/main/java/competition/subsystems/elevator/commands/TrimElevatorDown.java

```diff
@@ -14,12 +16,13 @@ public class TrimElevatorDown extends BaseCommand {
     public TrimElevatorDown(ElevatorSubsystem elevatorSubsystem, OperatorInterface operatorInterface){
         this.elevator = elevatorSubsystem;
         this.oi = operatorInterface;
-        this.addRequirements(elevator);
 
     }
 
     public void initialize(){
+        aKitLog.record("OldTrimValue", elevator.trimValue.get().in(Inches));
```

Comment:
⭐ this won't work quite the way you'd expect, for this kind of old/new logging lets use the text logger rather than the aKitLog
```suggestion
        var oldTrim = elevator.trimValue.get().in(Inches);
        elevator.trimElevatorDown();
        log.info("Updating trim value from " + oldTrim + " to " + elevator.trimValue.get().in(Inches));